README.md

```diff
@@ -415,7 +415,34 @@ Logger.getInstance().setLogLevel(Logger.LogLevel.Verbose);
   adb logcat > "C:\logmsg\logfile.txt"
  ```
  More examples about adb cmds: https://developer.android.com/tools/debugging/debugging-log.html#startingLogcat
- 
+
+#### Telemetry
```

Comment:
@iambmelt Daniel and I talked about that for MSAL, the documentation on diagnostics part should be in wiki, we can link the wiki on readme. I'm not sure if we want to do the same thing for ADAL, double check with him. If we do it in wiki, you can expand more on the default dispatcher and the aggregated one without overwhelming readme :)